pkg/networking/networking.go

```diff
@@ -65,12 +65,24 @@ func NewNetworkAccess(config configuration.Configuration) NetworkAccess {
 		logger.SetOutput(io.Discard)
 	}
 
-	return &networkImpl{
+	n := &networkImpl{
 		config:       config,
 		staticHeader: http.Header{},
 		logger:       logger,
 		proxy:        http.ProxyFromEnvironment,
 	}
+
+	extraCaCertFile := config.GetString(configuration.ADD_TRUSTED_CA_FILE)
+	if len(extraCaCertFile) > 0 {
+		err := n.AddRootCAs(extraCaCertFile)
+		if err != nil {
+			logger.Printf("Failed to AddRootCAs from '%s' (%v)\n", extraCaCertFile, err)
```

Comment:
Should this be a fatal error? If something is configured, but cannot be added, it feels like it should fail.

Reply:
I'm careful with exiting since it might be a too aggressive approach and might break things unexpectedly. I think step one is logging to increase transparency.